### PR TITLE
add dust coefficients for DECam bands uiY

### PR DESCRIPTION
**Pull request by moustakas**
Adds support for DR10 i-band and possible DR11 u- and Y-band.

### CODE REVIEW

**Review by weaverba137 — COMMENTED** [review]
I'd love to see additional test coverage as part of this PR.

File: py/desiutil/dust.py [hunk 1]
```diff
@@ -83,7 +96,7 @@ def extinction_total_to_selective_ratio(band, photsys, match_legacy_surveys=Fals
               'W3_S': 0.0241,
               'W4_S': 0.00910})
 
-    assert band.upper() in ["G", "R", "Z", "BP", "RP", 'W1', 'W2', 'W3', 'W4']
+    assert band.upper() in ["U", "G", "R", "I", "Z", "Y", "BP", "RP", 'W1', 'W2', 'W3', 'W4']
```
[the remark this answers is not in the document]
Comment (weaverba137) [reply py/desiutil/dust.py:99]: To be more precise, the function `extinction_total_to_selective_ratio()` is tested only with `match_legacy_surveys=False`.

I can see that the assert statements are a way to detect invalid inputs, but such an assertion error is not actually a useful message to the user of the function.  A `ValueError`, explaining the nature of the error should be raised instead.

**Comment by moustakas** [comment]
@weaverba137 I've decided to close this PR because the way the dust code is set is not flexible enough for what I'm doing in https://github.com/desihub/fastspecfit/pull/133. For example, in order to fit a general set of input photometry, which I'm extending `FastSpecFit` to be able to do, I need to be able to ignore the concept of a N/S photometric system (`photsys`) and I need to be able to have the same bandpasses from two different surveys (e.g., DECam g-band vs Subaru g-band). 

Since `desiutil.dust` is a key part of the pipeline, I decided it was easier for me to move the MW dust correction step over to `fastspecfit.util.mwdust_transmission`.

**Comment by weaverba137** [comment]
That's reasonable. Is it OK to delete the branch?

**Comment by moustakas** [comment]
I deleted the branch; thanks.

**Pull request closed without merging**